### PR TITLE
[wip] do not merge -- current diff for services

### PR DESCRIPTION
**Pull request by cjavilla-stripe**
cc @remi-stripe @ob-stripe 

Just for reference, here's the current diff when generating all services.

I'm going to work on ISingletonRetrievable and Custom service methods next.

### CODE REVIEW

File: src/Stripe.net/Services/Accounts/AccountService.cs [hunk 1]
```diff
@@ -26,6 +26,16 @@ public AccountService(IStripeClient client)
 
         public override string BasePath => "/v1/accounts";
 
+        public virtual ListAccountCapability Capabilities(string id, AccountCapabilitiesOptions options = null, RequestOptions requestOptions = null)
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Accounts/AccountService.cs]: Flagging this should not be here, we don't do nested on the parent service in dotnet for most cases except a handful. We might in the future, just not for this release

File: src/Stripe.net/Services/Invoices/InvoiceService.cs [hunk 2]
```diff
@@ -45,12 +46,12 @@ public virtual Task<Invoice> DeleteAsync(string id, RequestOptions requestOption
             return this.DeleteEntityAsync(id, null, requestOptions, cancellationToken);
         }
 
-        public virtual Invoice FinalizeInvoice(string id, InvoiceFinalizeOptions options = null, RequestOptions requestOptions = null)
+        public virtual Invoice FinalizeInvoice(string id, InvoiceFinalizeInvoiceOptions options = null, RequestOptions requestOptions = null)
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Invoices/InvoiceService.cs]: That one will need an override I think though I see why it used the full name

File: src/Stripe.net/Services/Recipients/RecipientService.cs [hunk 3]
```diff
@@ -0,0 +1,84 @@
+namespace Stripe
```
Comment (remi-stripe) [inline src/Stripe.net/Services/Recipients/RecipientService.cs]: that one will need to be removed

File: src/Stripe.net/Entities/CountrySpecs/CountrySpec.cs [hunk 4]
```diff
@@ -17,7 +17,10 @@ public class CountrySpec : StripeEntity<CountrySpec>, IHasId, IHasObject
         public string DefaultCurrency { get; set; }
 
         [JsonProperty("supported_bank_account_currencies")]
-        public Dictionary<string, List<string>> SupportedBankAccountCurrencies { get; set; }
+        public Dictionary<
```
Comment (cjavilla-stripe) [inline src/Stripe.net/Entities/CountrySpecs/CountrySpec.cs]: interesting.

File: src/Stripe.net/Entities/Invoices/Invoice.cs [hunk 5]
```diff
@@ -109,7 +93,7 @@ public Customer Customer
         public string CustomerPhone { get; set; }
 
         [JsonProperty("customer_shipping")]
-        public Shipping CustomerShipping { get; set; }
+        public CustomerShipping CustomerShipping { get; set; }
```
Comment (cjavilla-stripe) [inline src/Stripe.net/Entities/Invoices/Invoice.cs]: will need to override this

**Pull request closed without merging**